sql/sqlite/driver.go

```diff
@@ -99,6 +105,43 @@ func (d *Driver) Clean(ctx context.Context) error {
 	return nil
 }
 
+// Lock implements the schema.Locker interface.
+func (d *Driver) Lock(_ context.Context, name string, timeout time.Duration) (schema.UnlockFunc, error) {
+	dir, err := os.UserCacheDir()
```

Comment:
Why the elaborate why do you use CacheDir and not TempDir for example?

Reply:
TempDir might get cleaned up occasionally by OS and I don't want that. Cache will live at least until reboot. And in that case it doesn't matter anymore since the DB will rebooted as well.

Reply:
afaik, temporary files won’t be deleted until the process is exit. Am I wrong?